index.js

```diff
@@ -0,0 +1,154 @@
+function toggle(spanElement, arrowId) {
+  var arrow = document.getElementById(arrowId);
+
+  if (arrow.classList.contains("bi-caret-down-fill")) {
+    arrow.classList.remove("bi-caret-down-fill");
+    arrow.classList.add("bi-caret-up-fill");
+  } else {
+    arrow.classList.remove("bi-caret-up-fill");
+    arrow.classList.add("bi-caret-down-fill");
+  }
+}
+
+function filterFood() {
+  const priceRange = uislider.noUiSlider.get();
+  const minPrice = parseFloat(priceRange[0]);
+  const maxPrice = parseFloat(priceRange[1]);
+
+  const foodItems = document.getElementsByClassName("food-item");
+
+  for (let i = 0; i < foodItems.length; i++) {
+    const foodPriceElement = foodItems[i].querySelector(".food-price");
+    const foodPrice = parseFloat(
+      foodPriceElement.textContent.replace("Price: $", "")
```

Comment:
one nice way to avoid this (since it may become a problem if the website is in multiple languages for example) is to use data attributes. You could add a `data-price` attribute that it's a float to your dom element and use it here. In this way you won't need to extract the price from a longer string.